docs/2_step_by_step/2_3_login_to_jina_ecosystem.md

```diff
@@ -1 +1,27 @@
-# Login to Jina ecosystem
\ No newline at end of file
+# Login to Jina ecosystem
+
+Since Finetuner leverages cloud resources for Fine-tuning.
+Before starting a fine-tuning job,
+it is required to login and obtain a token from Jina.
+It is as simple as:
+
+```python
+import finetuner
+
+finetuner.login()
+```
+
+A browser window should pop-up with different login options,
+After login you will see this in your terminal:
```

Comment:
Maybe it would be helpful to add where the token is stored. Otherwise, it is a bit confusing, since you mentioned there is a token, but it is not clear where it is. Alternatively, I would suggest to just leaf out that there is a token at all.